scripting/get5.sp

```diff
@@ -686,35 +686,57 @@ public Action Timer_CheckReady(Handle timer) {
 
 static void CheckReadyWaitingTimes() {
   if (g_TeamTimeToStartCvar.IntValue > 0) {
-    CheckReadyWaitingTime(MatchTeam_Team1);
-    CheckReadyWaitingTime(MatchTeam_Team2);
+    bool team1Forfeited;
+    bool team2Forfeited;
+
+    g_ReadyTimeWaitingUsed++;
+
+    team1Forfeited = CheckReadyWaitingTime(MatchTeam_Team1);
+    team2Forfeited = CheckReadyWaitingTime(MatchTeam_Team2);
+
+    if (team1Forfeited && team2Forfeited)
+    {
```

Comment:
style nitpick: to be consistent with the rest of the code, can you format this in the 
```
if (...) {
  ...
} else if (...) {
  ...
}
```

style?